cmd/nf-dump-pcap/main.go

```diff
@@ -15,14 +15,14 @@ import (
 
 	"github.com/google/gopacket"
```

Comment:
I'm pretty sure it's going to get imported by Jenkins when we run `go test ...` - if we don't use the `cmd` directory, can you remove it?